type-tests/files/createAction.typetest.ts

```diff
@@ -156,6 +156,8 @@ function expectType<T>(p: T): T {
 
   // typings:expect-error
   expectType<string>(strLenAction('test').payload)
+  // typings:expect-error
+  expectType<boolean>(strLenAction('test').error)
```

Comment:
Please replace this with something like  
```typescript
// typings:expect-error
const x = strLenMetaAction('test').error
```
you want to see that the property does not exist at all, while the 
```typescript
// typings:expect-error 
expectType<boolean>(...)
```
 just ensures that it isn't boolean.

Reply:
I updated the test as suggested.
Out of curiosity, would it be possible to add Jest expectations here?
```ts
  // typings:expect-error
  const error: any = strLenAction('test').error
  expect(error).toBeUndefined()
```

Reply:
I don't think that code is actually ever run, it should only be compiled to look for compiler warnings. Doing something like this would work better in the normal tests.

We just want to make sure that there's no regression in the types & autocompletion.